app/src/main/java/com/connfa/ui/fragment/EventHolderFragment.java

```diff
@@ -159,83 +158,47 @@ private void initView() {
+        viewPager = (ViewPager) view.findViewById(R.id.viewPager);
+        viewPager.setAdapter(adapter);
 
         Typeface typeface = Typeface.createFromAsset(getActivity().getAssets(), "fonts/Roboto-Regular.ttf");
-        mPagerTabs = (PagerSlidingTabStrip) getView().findViewById(R.id.pager_tab_strip);
-        mPagerTabs.setTypeface(typeface, 0);
-        mPagerTabs.setViewPager(mViewPager);
+        tabStrip = (PagerSlidingTabStrip) getView().findViewById(R.id.pager_tab_strip);
+        tabStrip.setTypeface(typeface, 0);
+        tabStrip.setViewPager(viewPager);
 
-        mLayoutPlaceholder = view.findViewById(R.id.layout_placeholder);
-        mTextViewNoContent = (TextView) view.findViewById(R.id.text_view_placeholder);
-        mImageViewNoContent = (ImageView) view.findViewById(R.id.image_view_placeholder);
+        layoutPlaceholder = view.findViewById(R.id.layout_placeholder);
+        emptyLabel = (TextView) view.findViewById(R.id.text_view_placeholder);
+        emptyImageView = (ImageView) view.findViewById(R.id.image_view_placeholder);
 
-        if (mEventMode == DrawerManager.EventMode.PROGRAM ||
-                mEventMode == DrawerManager.EventMode.BOFS ||
-                mEventMode == DrawerManager.EventMode.SOCIAL) {
+        if (eventMode == DrawerManager.EventMode.PROGRAM ||
+                eventMode == DrawerManager.EventMode.BOFS ||
+                eventMode == DrawerManager.EventMode.SOCIAL) {
             setHasOptionsMenu(true);
         } else {
             setHasOptionsMenu(false);
         }
     }
 
-    class LoadData extends AsyncTask<Void, Void, List<Long>> {
-
-        @Override
-        protected List<Long> doInBackground(Void... params) {
-            return getDayList();
-        }
-
-        @Override
-        protected void onPostExecute(List<Long> result) {
-            updateViews(result);
-        }
-    }
-
-    private List<Long> getDayList() {
-        List<Long> dayList = new ArrayList<>();
-        switch (mEventMode) {
-            case BOFS:
-                BofsManager bofsManager = Model.getInstance().getBofsManager();
-                dayList.addAll(bofsManager.getBofsDays());
-                break;
-            case SOCIAL:
-                SocialManager socialManager = Model.getInstance().getSocialManager();
-                dayList.addAll(socialManager.getSocialsDays());
-                break;
-            case FAVORITES:
-                FavoriteManager favoriteManager = Model.getInstance().getFavoriteManager();
-                dayList.addAll(favoriteManager.getFavoriteEventDays());
-                break;
-            default:
-                ProgramManager programManager = Model.getInstance().getProgramManager();
-                dayList.addAll(programManager.getProgramDays());
-                break;
-        }
-        return dayList;
-    }
-
     private void updateViews(List<Long> dayList) {
 
 //        if(!isResumed()){
```

Comment:
@frapontillo was for some reason against that in the past. I'll gladly kill it with fire.